package.json

```diff
@@ -2,19 +2,28 @@
   "name": "atom-select-list",
   "version": "0.7.2",
   "description": "A general-purpose select list for use in Atom packages",
-  "main": "./src/select-list-view.js",
+  "main": "lib/select-list-view.js",
+  "files": [
+    "lib/**/*"
+  ],
   "scripts": {
-    "test": "atom --test test"
+    "typescript": "tsc -p src/tsconfig.json || echo 'tsc is done'",
+    "build": "npm run typescript -s",
```

Comment:
Can we just make this `tsc -p src/tsconfig.json` and not worry about hiding the output?

Reply:
I changed it in [this commit](https://github.com/atom/atom-select-list/pull/28/commits/a37549cde6322cd0d0745811a0e58a0b548f8358)

However, this means that we need to update the source code, which I wanted to avoid so it gets merged faster.